subworkflows/local/profiling.nf

```diff
@@ -39,7 +41,7 @@ workflow PROFILING {
             .combine(databases)
             .branch {
                 malt:    it[2]['tool'] == 'malt'
-                kraken2: it[2]['tool'] == 'kraken2'
+                kraken2: it[2]['tool'] == 'kraken2' || it[2]['tool'] == 'bracken'
```

Comment:
Would this then mean that even non-bracken kraken2 databases would be attempted to be executed by bracken?